.github/workflows/platformDeploy.yml

```diff
@@ -420,7 +424,7 @@ jobs:
   postGithubComment:
     name: Post a GitHub comment when platforms are done building and deploying
     runs-on: ubuntu-latest
-    if: ${{ always() }}
+    if: ${{ always() && !cancelled() }}
```

Comment:
```suggestion
    if: ${{ !cancelled() }}
```

Reply:
NAB - this suggestion seems correct to me, since always will always be true.